src/components/TodoList.jsx

```diff
@@ -50,6 +50,17 @@ export function TodoList() {
     setTodos(newTodos);
     setTodos(newId);
   };
+  const toggleStatus = (todoId) => {
+    const toggleButton = todos.map((todo) => {
+      if (todo.id === todoId)
+        return {
+          ...todo,
+          status: todo.status === "作業中" ? "完了" : "作業中",
+        };
```

Comment:
このようにifを使うときは`{ }`で囲うのが適切です

```suggestion
      if (todo.id === todoId) {
        return {
          ...todo,
          status: todo.status === "作業中" ? "完了" : "作業中",
        };
      }
```

Reply:
了解しました